### PR TITLE
Webcomponent header

### PR DESCRIPTION
**Pull request by f-necas**
# Webcomponent header

Needs resolution of PR : 
- https://github.com/georchestra/geonetwork-ui/pull/7

Implements the new georchestra/header in datafeeder-ui.
Main PR : 
- https://github.com/georchestra/georchestra/pull/4065

### Header script
Old iframe header is removed from the image and can be available while with an `sh` script (e.g. use iframe header): 
``` sh
DATAFEEDER_DIR=${1:-/usr/share/nginx/html/}
SNIPPET="<script src='https://cdn.jsdelivr.net/gh/georchestra/header@dist/header.js'></script><geor-header active-app='import' style='height:90px' legacy-header="true" legacy-url="/header/"></geor-header>"

if grep -q "${SNIPPET}" "${DATAFEEDER_DIR}/index.html"; then
  echo "[INFO] geOrchestra: header already present."
  exit 0
fi

echo "[INFO] geOrchestra: adding header in the main page..."
sed -i "s#<body class=\"m-0 h-full flex flex-col\">#<body class=\"m-0 h-full flex flex-col\">${SNIPPET}#" ${DATAFEEDER_DIR}/index.html

```
For all informations about the header : see [README.md](https://github.com/georchestra/header)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ FROM nginx:1.21.4-alpine
 RUN rm -rf /usr/share/nginx/html/*
 COPY dist/apps/datafeeder /usr/share/nginx/html
 COPY nginx-default.conf /etc/nginx/conf.d/default.conf
+COPY custom-start.sh /custom-start.sh
 EXPOSE 80
 
+ENTRYPOINT ["sh", "/custom-start.sh", "sh", "/docker-entrypoint.sh"]
```
Comment (tkohr) [inline Dockerfile]: I guess `"/docker-entrypoint.sh"` refers to the other PR. But why do we need both scripts `/custom-start.sh` and `/docker-entrypoint.sh`?

**Review by tkohr — COMMENTED** [review]
Thanks @f-necas ! Just left a question for clarification.

And I guess there will be another linked PR to the datadir to add the script that adds the new header similar to the one you put in the description with the old header?

**Comment by edevosc2c** [comment]
Please rebase against master :).

**Comment by tkohr** [comment]
I think master is not up-to-date/relevant in this repo. So just needs a rebase on `georchestra-datafeeder`.

**Comment by f-necas** [comment]
I rebased it on `georchestra-datafeeder` to be more relevant. It was an old thing. 

@tkohr if you can review it again :)

**Review by tkohr — APPROVED** [review]
Thanks @f-necas changes LGTM.

Just didn't understand why you added `<link href="https://fonts.googleapis.comicon?family=Material+Icons|Material+Icons+Outlined" rel="stylesheet"/>` to the `index.html` in your previous commit, but not here anymore.

**Pull request merged**